src/test/java/seedu/giatros/testutil/PatientUtil.java

```diff
@@ -37,6 +39,11 @@ public static String getPatientDetails(Patient patient) {
         patient.getAllergies().stream().forEach(
             s -> sb.append(PREFIX_ALLERGY + s.allergyName + " ")
         );
+        // ! temporary fix, need space or in test we get y/apt/
```

Comment:
What is the issue here?

Reply:
you'll end up with an issue where you have `y/apt/` in the case that allergy is null. this leads to the interpretation that allergy is `apt/`